web/impact/impact/settings.py

```diff
@@ -54,6 +54,8 @@ class Base(Configuration):
 
     DATABASE_ROUTERS = ['impact.routers.APIRouter']
 
+    SLAVE_DATABASES = ['default']
```

Comment:
This is the right way to reference the `'default'` database alias, but I don't see how that alias is getting set in the main `DATABASES` configuration above. In the docs for [dj-database-url](https://pypi.python.org/pypi/dj-database-url/#usage) (which [django-configurations' DatabaseURL method](http://django-configurations.readthedocs.io/en/latest/values/#configurations.values.DatabaseURLValue) uses), the alias is assigned explicitly. Is that happening automatically here in some way I'm missing?

Reply:
this value is being set just to make the library happy when imported locally (in fact in reality we could omit a value here altogether). the django database url deserializes to set the database 'default' by default as well.

Reply:
also we are guarenteed to have a 'default' database since its required as per the django docs: https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DATABASES
```The DATABASES setting must configure a default database;```

Reply:
After discussing with Shankar i'm satisfied that  django-configurations' DatabaseURL method returns a full configuration dict, complete with a `default` entry. Approving.